src/NzbDrone.Core/Download/Clients/Deluge/DelugeProxy.cs

```diff
@@ -103,6 +103,9 @@ public string AddTorrentFromMagnet(string magnetLink, DelugeSettings settings)
         {
             var options = new
                           {
+                              download_location = settings.DownloadDirectory,
+                              move_completed_path = settings.CompletedDirectory,
+                              move_completed = settings.CompletedDirectory.IsNotNullOrWhiteSpace(),
```

Comment:
Will this cause issues if the user have configured something on the deluge side to move completed torrents to a different directory even if they haven't configured `download_location` or `move_completed_path`?

Reply:
You make a good point. If the client fields aren't filled in on Sonarr Deluge Client settings, they shouldn't be sent to Deluge. I've added in some logic in a new commit so the options will only get passed to Deluge if they are actually configured in Sonarr:

https://github.com/Sonarr/Sonarr/blob/dd73763bb937c768d20ced2cffd102d5e2257ae3/src/NzbDrone.Core/Download/Clients/Deluge/DelugeProxy.cs#L104-L119